files/zh-cn/web/http/status/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: HTTP 响应状态码
+slug: Web/HTTP/Status
+tags:
+  - HTTP
+  - Landing
+  - Overview
+  - Reference
+  - 状态码
+  - Web
+---
+{{HTTPSidebar}}
+
+HTTP 响应状态码用来表明特定 [HTTP](/en-US/docs/Web/HTTP) 请求是否成功完成。
+响应被归为以下五大类:
+
+1. [信息响应](#information_responses) (`100`–`199`)
+2. [成功响应](#successful_responses) (`200`–`299`)
+3. [重定向消息](#redirection_messages) (`300`–`399`)
+4. [客户端错误响应](#client_error_responses) (`400`–`499`)
+5. [服务端错误响应](#server_error_responses) (`500`–`599`)
```

Comment:
those anchors are not correct.

```md
1. [信息响应](#信息响应) (`100`–`199`)
2. [成功响应](#成功响应) (`200`–`299`)
3. [重定向消息](#重定向消息) (`300`–`399`)
4. [客户端错误响应](#客户端错误响应) (`400`–`499`)
5. [服务端错误响应](#服务端错误响应) (`500`–`599`)
```

In markdown, the head is almost the same as the `id`.

Reply:
so, please redo the commits and also update those anchors.